README.md

```diff
@@ -5,11 +5,1119 @@
+```
+
+And get the resulting GraphQL API:
+
+<details>
+<summary>(toggle me)</summary>
+
+```gql
+input IntFilter {
+  equals: Int
+  gt: Int
+  gte: Int
+  in: [Int!]
+  lt: Int
+  lte: Int
+  not: Int
+  notIn: [Int!]
+}
+
+type Mutation {
+  createOnePost(data: PostCreateInput!): Post!
+  createOneUser(data: UserCreateInput!): User!
+}
+
+enum OrderByArg {
+  asc
+  desc
+}
+
+type Post {
+  author(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    skip: Int
+  ): [User!]
+  id: Int!
+}
+
+input PostCreateInput {
+  author: UserCreateManyWithoutAuthorInput
+}
+
+input PostCreateManyWithoutPostsInput {
+  connect: [PostWhereUniqueInput!]
+  create: [PostCreateWithoutAuthorInput!]
+}
+
+input PostCreateWithoutAuthorInput
+
+input PostFilter {
+  every: PostWhereInput
+  none: PostWhereInput
+  some: PostWhereInput
+}
+
+input PostWhereInput {
+  AND: [PostWhereInput!]
+  author: UserFilter
+  id: IntFilter
+  NOT: [PostWhereInput!]
+  OR: [PostWhereInput!]
+}
+
+input PostWhereUniqueInput {
+  id: Int
+}
+
+type Query {
+  post(where: PostWhereUniqueInput!): Post
+  posts(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    skip: Int
+    where: PostWhereInput
+  ): [Post!]
+  user(where: UserWhereUniqueInput!): User
+  users(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    orderBy: UserOrderByInput
+    skip: Int
+  ): [User!]
+}
+
+input StringFilter {
+  contains: String
+  endsWith: String
+  equals: String
+  gt: String
+  gte: String
+  in: [String!]
+  lt: String
+  lte: String
+  not: String
+  notIn: [String!]
+  startsWith: String
+}
+
+type User {
+  handle: String!
+  id: ID!
+  posts(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    skip: Int
+  ): [Post!]
+}
+
+input UserCreateInput {
+  handle: String!
+  id: ID
+  posts: PostCreateManyWithoutPostsInput
+}
+
+input UserCreateManyWithoutAuthorInput {
+  connect: [UserWhereUniqueInput!]
+  create: [UserCreateWithoutPostsInput!]
+}
+
+input UserCreateWithoutPostsInput {
+  handle: String!
+  id: ID
+}
+
+input UserFilter {
+  every: UserWhereInput
+  none: UserWhereInput
+  some: UserWhereInput
+}
+
+input UserOrderByInput {
+  handle: OrderByArg
+  id: OrderByArg
+}
+
+input UserWhereInput {
+  AND: [UserWhereInput!]
+  handle: StringFilter
+  id: StringFilter
+  NOT: [UserWhereInput!]
+  OR: [UserWhereInput!]
+  posts: PostFilter
+}
+
+input UserWhereUniqueInput {
+  id: ID
+}
+```
+
+</details>
+
+<br>
+
+You can find a runnable version of this and other examples at [prisma-labs/nexus-examples](/TODO).
+
+## Reference
+
+### `t.model`
+
+`t.model` is for projecting your models’ fields onto your `Object`s. It is only available inside `Object` definitions. `t.model` properties are called "Field Projectors" and there is one per field on your respective model. They accept configuration allowing you to control how the model's fields are seen through your GraphQL schema.
```

Comment:
> I'm a bit confused by the inline code formatting for Object? are you referring to objectType here? I think for inline code formatting, we should try to apply it only to concepts that are literally code pieces, so if Object stands for objectType here, maybe indeed use objectType instead?

It is the formal (spec) GraphQL Type name. Along with `Enum` `InputObject` `Union` and so on. A verbose way of saying it might be "GraphQL `Object`". Situations like this might be reason for us to create a small legend/intro/glossary.

Here is a before/after comparison:

Before:

![image](https://user-images.githubusercontent.com/284476/66659763-12c55700-ec12-11e9-9a4d-b0ee532bcafa.png)

After:

![image](https://user-images.githubusercontent.com/284476/66659844-312b5280-ec12-11e9-97ed-a31270e2181a.png)

Will think about this more!

> Question: Are we sure we want to uppercase concepts like field projectors? To me these are regular features (not names) so I'd rather lowercase them. I've been in that situation a number of times when documenting Prisma features, and I feel like defaulting to lowercasing generally feels more natural. Otherwise it's hard to draw a boundary when something gets lowercased and when it gets uppercased.

Yeah not sure. Will think about this more!

> Second note: I'd write this in italics instead of "quotes". 

👍